### PR TITLE
Fix: Modify spacing between title, description, and CTA button

### PR DESCRIPTION
**Pull request by alain-kubwayo**
## Describe your changes
Changed spacing between title and description and reduced gap between `schedule call` button and description on mobile screen
## Issue ticket number and link
- ID: 067
- Link: [Notion](https://www.notion.so/apeunit/Fix-ContactUs-section-f5e8a266f660441db2f686d72028c68e)

## Tasks completed
- [x] Change spacing between title and description
- [x] Reduce gap between `schedule call` button and description on mobile screen
 
## Tasks not completed
- None

### CODE REVIEW

File: components/ui/Typography.jsx [hunk 1]
```diff
@@ -25,6 +25,6 @@ export const Heading4 = ({ children, className }) => {
 
 export const Paragraph = ({ children, className }) => {
     return (
-        <p className={`font-normal leading-4.95 md:leading-7.5 text-4.5 md:text-6 tracking-0.015 ${className}`}>{children}</p>
+        <p className={`font-normal leading-4.95 md:leading-7.5 text-4.5 md:text-6 -tracking-0.015 ${className}`}>{children}</p>
```
Comment (Ntwali-Josue) [inline components/ui/Typography.jsx:28]: - Pull changes from the origin `dev` branch to have the updated `Paragraph` component.

**Review by Ntwali-Josue — CHANGES_REQUESTED** [review]
Hey @alain-kubwayo, great work on implementing the fixes. I have highlighted a few changes to improve and your PR will be good to be approved. Check them down below:

File: components/sections/ContactUs.jsx [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import LargeButton from "../ui/buttons/LargeButton";
-import { Heading2 } from "../ui/Typography";
+import { Heading2, Paragraph } from "../ui/Typography";
 
 const ContactUs = () => {
     return (
-        <section className="bg-primary-800 mx-0.5 rounded-8 gap-16 xl:px-0 px-5 py-5.5 md:py-16 flex flex-col items-center">
+        <section className="bg-primary-800 mx-0.5 rounded-8 gap-8 md:gap-16 xl:px-0 px-5 py-5.5 md:py-16 flex flex-col items-center">
             <div className="xl:w-3.73/5">
                 <Heading2 className="text-center text-white text-11.75">Ready to talk about your project?</Heading2>
```
[the remark this answers is not in the document]
Comment (alain-kubwayo) [reply components/sections/ContactUs.jsx:8]: Thanks for catching that.

File: components/sections/ContactUs.jsx [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-                    <p className="text-center text-white/90">Are you looking for a consultancy partnership to address your blockchain challenges? We&apos;d love to hear from you.  Let&apos;s build something</p>
-                    <p className="text-center text-white/90">amazing together.</p>
-                </div>
+                <Heading2 className="text-center">Ready to talk about your project?</Heading2>
+                <Paragraph className="mt-3 text-center text-white">
+                    Are you looking for a consultancy partnership to address your blockchain challenges?
+                    <br className="hidden md:block" />We&apos;d love to hear from you. Let&apos;s build something amazing together.
+                </Paragraph>
             </div>
             <LargeButton text="Schedule a call" styleSpan="mt-0.13" hasArrow={true} redirectLink="mailto:u2467@apeunit.com" />
```
Comment (mhenriette) [inline components/sections/ContactUs.jsx:14]: the button text should have a font size of  20px on the mobile screens , i suggest you use the right component of **Heading4**

**Review by mhenriette — CHANGES_REQUESTED** [review]
@alain-kubwayo great Job on the changes made. however, I left a small comment, please check it out.

File: components/ui/Typography.jsx [hunk 4]
```diff
@@ -25,6 +25,6 @@ export const Heading4 = ({ children, className }) => {
 
 export const Paragraph = ({ children, className }) => {
     return (
-        <p className={`font-normal leading-6.75 md:leading-7.5 text-4.5 md:text-6 tracking-0.015 ${className}`}>{children}</p>
+        <p className={`font-normal leading-6.75 md:leading-7.5 text-4.5 md:text-6 -tracking-0.015 ${className}`}>{children}</p>
```
Comment (Ntwali-Josue) [inline components/ui/Typography.jsx:28]: Good job on fixing this `tracking` 💯

**Review by Ntwali-Josue — APPROVED** [review]
Great work @alain-kubwayo

**Review by mhenriette — APPROVED** [review]
Looks Good!!

**Pull request merged**